priv/module_static/lib/runes.js

```diff
@@ -6407,7 +6413,7 @@
   getReadyWebGLForCanvas('3d');
 
   exports.anaglyph = anaglyph;
-  exports.animate = animate;
+  exports.animated_show = animated_show;
```

Comment:
How can I test this?

Reply:
You need a journey-frontend to test this. The whole new lib/runes requires to be run on the journey-frontend.